browser/sources.gni

```diff
@@ -323,6 +321,8 @@ if (is_android) {
     "//brave/browser/brave_ads/tooltips",
     "//brave/browser/importer",
     "//brave/browser/ui/bookmark",
+    "//brave/components/ai_chat",
+    "//brave/components/ai_chat:mojom",
```

Comment:
Including `//brave/components/ai_chat` will have constants.{h|cc} included which has localized strings. Do we want that?

Reply:
Makes sense.